2-farm/translations/Readme.hi.md

```diff
@@ -0,0 +1,22 @@
+# आई.ओ.टी के साथ खेती
+
+जैसे-जैसे जनसंख्या बढ़ती है, वैसे-वैसे कृषि की मांग भी बढ़ती है। मगर उपलब्ध भूमि की मात्रा नहीं बदलती है, लेकिन जलवायु बदलती है जो याहा किसानों को और भी अधिक चुनौतियाँ देती है, विशेषकर 2 बिलियन [निर्वाह किसान](https://wikipedia.org/wiki/Subsistence_agriculture) जो इस बात पर भरोसा करते हैं कि वे क्या उगाते हैं अपने परिवार का भरण-पोषण करने के लिए। आई.ओ.टी किसानों को बेहतर निर्णय लेने में मदद कर सकता है कि क्या उगाया जाए और कब कटाई की जाए, पैदावार बढ़ाई जाए, शारीरिक श्रम की मात्रा को कम किया जाए और कीटों का पता लगाया जाए और उनसे निपटा जा ske।
+
+इन 6 पाठों में आप सीखेंगे कि खेती को बेहतर बनाने और स्वचालित करने के लिए इंटरनेट ऑफ थिंग्स को कैसे लागू किया जाए।
+
+> 💁 ये पाठ कुछ क्लाउड संसाधनों का उपयोग करेंगे। यदि आप इस परियोजना के सभी पाठों को पूरा नहीं करते हैं, तो सुनिश्चित करें कि आप [अपनी परियोजना को साफ करें](पाठ/6-कीप-योर-प्लांट-सिक्योर/रीडएमई.एमडी#क्लीन-अप-योर-प्रोजेक्ट) चरण का पालन करें। [पाठ ६] में (पाठ/6-कीप-योर-प्लांट-सिक्योर/रीडएमई.एमडी)।
+
+**स्वचालित संयंत्र का वीडियो जोड़ें**
```

Comment:
Looks like you are translating from an out of data version of the repo - this note was removed a while ago.

Reply:
I am sorry I didn't see that I have updated this file